src/util/logger.js

```diff
@@ -0,0 +1,24 @@
+import configuration from '../configuration.js';
+
+// MessageSeverity definition
+const MessageSeverity = {};
+
+Object.defineProperties(MessageSeverity, {
+  'info': { enumerable: true, writable: false, value: 0 },
+  'warning': { enumerable: true, writable: false, value: 1 },
+  'error': { enumerable: true, writable: false, value: 2 },
+  'exception': { enumerable: true, writable: false, value: 3 }
+});
+
+export { MessageSeverity };
+
+// Logger definition
+function log (message, messageSeverity) {
```

Comment:
It would be nice if we could pass multiple parameters to log. Now that Tim added his pretty printer, we can rely on console.log to pretty print multiple args to logcat. The easiest way is probably to take the severity argument first and use a rest parameter for the others:

```javascript
function logWithSeverity(severity, ...messageArgs) {
  if (severity >= configuration.logMinMessageSeverity) {
    console.log(...messageArgs);
  }
}
```

Reply:
Great idea @tonycuadra !